### PR TITLE
Update testing packges

### PR DESCRIPTION
**Pull request by dopplershift**
Move to 2.2.0. Also let's get these working on windows and python 3.

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -11,5 +11,5 @@ ECHO toolkits_tests = False >> setup.cfg
 ECHO [test] >> setup.cfg
 ECHO local_freetype = True >> setup.cfg
 
-%PYTHON% setup.py install --single-version-externally-managed --record record.txt
+%PYTHON% -m pip install -v
```
Comment (ocefpaf) [inline recipe/bld.bat]: I believe that `python -m pip install --no-deps --ignore-installed .` is safer.

Comment (dopplershift) [reply recipe/bld.bat]: Seems fair. I'm just trying to get the damned thing to work.

**Comment by dopplershift** [comment]
Power cycling to try to get AppVeyor to wake up.

**Comment by dopplershift** [comment]
Closed in favor of #151 .

**Pull request closed without merging**